import-export-cli/cmd/mi/delete/user.go

```diff
@@ -0,0 +1,75 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package delete
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/credentials"
+	"github.com/wso2/product-apim-tooling/import-export-cli/mi/impl"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+var deleteUserCmdEnvironment string
+
+const deleteUserCmdLiteral = "user [user-name]"
+const deleteUserCmdShortDesc = "Delete a user from the Micro Integrator"
+
+const deleteUserCmdLongDesc = "Delete a user with the name specified by the command line argument [user-name] from a Micro Integrator in the environment specified by the flag --environment, -e"
+
+var deleteUserCmdExamples = "To delete a user\n" +
+	"  " + utils.ProjectName + " " + utils.MiCmdLiteral + " " + deleteCmdLiteral + " user user2 -e dev\n" +
+	"NOTE: The flag (--environment (-e)) is mandatory"
+
+var deleteUserCmd = &cobra.Command{
+	Use:     deleteUserCmdLiteral,
+	Short:   deleteUserCmdShortDesc,
+	Long:    deleteUserCmdLongDesc,
+	Example: deleteUserCmdExamples,
+	Args:    cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		handledeleteUserCmdArguments(args)
+	},
+}
+
+func init() {
+	DeleteCmd.AddCommand(deleteUserCmd)
+	deleteUserCmd.Flags().StringVarP(&deleteUserCmdEnvironment, "environment", "e", "", "Environment to be searched")
+	deleteUserCmd.MarkFlagRequired("environment")
+}
+
+func handledeleteUserCmdArguments(args []string) {
+	printDeleteCmdVerboseLog(deleteUserCmdLiteral)
+	credentials.HandleMissingCredentials(deleteUserCmdEnvironment)
+	executeDeleteUser(args[0])
+}
+
+func executeDeleteUser(userName string) {
+	resp, err := impl.DeleteMIUser(deleteUserCmdEnvironment, userName)
+	if err != nil {
+		fmt.Println(utils.LogPrefixError+"deleteing user [ "+userName+" ]", err)
+	} else {
+		fmt.Println("Deleteing user [ "+userName+" ] status:", resp)
```

Comment:
Shall we change this as follows?
```suggestion
		fmt.Println("Deleting user [ "+userName+" ] status:", resp)
```

Reply:
Fixed with 3e012c9